### PR TITLE
Add tape usage information to server api

### PR DESCRIPTION
**Pull request by sjaakieb**
For some tests it is necessary to check if a tape has been used.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -95,6 +95,14 @@ export default class TalkbackServer {
     return this.server
   }
 
+  hasTapeBeenUsed(tapeName) {
+    return this.tapeStore.tapes.some(t => t.used && t.path === tapeName);
```
Comment (ijpiantanida) [inline src/server.js]: I'd delegate the implementation to `tapeStore`.

File: src/server.js [hunk 2]
```diff
@@ -95,6 +95,14 @@ export default class TalkbackServer {
...
+  }
+
+  resetTapeUsage() {
```
Comment (ijpiantanida) [inline src/server.js]: Same as previous comment, delegate the implementation to `tapeStore`.

**Comment by sjaakieb** [comment]
Our usecase is :
- Login
- Get a list of objects
- Create a new object
- Get back to the list of objects
- Assert the new object has been created.

The last assert is where I need to check if a POST call has been made.

**Comment by ijpiantanida** [comment]
@sjaakieb v1.6.0 was just released with your changes included.

Thank you for the contribution!

**Pull request merged**